utils/resource-monitor/auto.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+###################################################
+# Automates batch job resource graphing.
+###################################################
+
+# This script:
+# Takes in a lower bound, upper bound and increment for the number of cores, and graphs resource usage for each run.
+
+# Example usage:
+# sh auto.sh [data description] [output parquet path] [output results path] [num processes increment] [lower core limit] [upper core limit]
+
```

Comment:
Using `set -e` in general is a good idea for bash script, unless there are good reasons not to.